tests/apollo/test_skvbc_checkpoints.py

```diff
@@ -440,7 +441,7 @@ async def test_rvt_conflict_detection_after_corrupting_checkpoint_msg_for_f_plus
             with self.assertRaises(trio.TooSlowError):
                   await skvbc.fill_and_wait_for_checkpoint(
                     initial_nodes=bft_network.all_replicas(without=byzantine_replica_ids),
-                    num_of_checkpoints_to_add=5,
+                    num_of_checkpoints_to_add=2,
```

Comment:
Please correct me on below claim, I might be missing something again...

I am not convinced that this is what we want to check, even with the new behavior.
I think we still want to try to advance 5 checkpoints, but see that network advanced only 2 checkpoints.
Otherwise, what are you testing here? The idea is to show that network does not advance out of the window ,since there are f+1 byzantine. If you write only 2 checkpoints, they will all reach that point, since it's still inside the window. In that case you are loosing the whole idea of the test, test always succeed and nothing was tested.Better way is to:
1) write 5 checkpoints
2) check that network advanced only 2
3) check that number_of_checkpoint_mismatch == (f+1)*5

Reply:
Thanks Gil.
I agree with first two points.
Third point is not feasible as at max there are going to be only (f+1)*2 mismatched checkpoint.

Reply:
Oh ye, you are right..